src/sonic-config-engine/tests/sample_output/py3/qos-arista7800r3-48cq2-lc.json

```diff
@@ -115,7 +115,7 @@
     },
     "PORT_QOS_MAP": {
         "global": {
-            "dscp_to_tc_map"  : "AZURE"
+            "dscp_to_tc_map"  : "[DSCP_TO_TC_MAP|AZURE]"
         },
         "Ethernet0": {
             "dscp_to_tc_map"  : "AZURE",
```

Comment:
Please change all the outputs in this file to use the reference format

Reply:
Updated. Thanks